flood_adapt/integrator/sfincs_adapter.py

```diff
@@ -157,28 +691,27 @@ def add_wl_bc(self, df_ts: pd.DataFrame):
             Dataframe with waterlevel time series at every boundary point (index of the dataframe should be time and every column should be an integer starting with 1)
         """
         # Determine bnd points from reference overland model
-        gdf_locs = self.sf_model.forcing["bzs"].vector.to_gdf()
-        gdf_locs.crs = self.sf_model.crs
+        gdf_locs = self._model.forcing["bzs"].vector.to_gdf()
+        gdf_locs.crs = self._model.crs
 
         if len(df_ts.columns) == 1:
             # Go from 1 timeseries to timeseries for all boundary points
             for i in range(1, len(gdf_locs)):
                 df_ts[i + 1] = df_ts[1]
 
         # HydroMT function: set waterlevel forcing from time series
-        self.sf_model.set_forcing_1d(
+        self._model.set_forcing_1d(
             name="bzs", df_ts=df_ts, gdf_locs=gdf_locs, merge=False
         )
 
-    def add_bzs_from_bca(
+    def _add_bzs_from_bca(
```

Comment:
this is for us only relevant for the offshore model: this generates the tide only at the SFINCS offshore model boundary points. We need a specific function because the offshore model is large and has several boundary points. As the tidal wave propagates through the offshore model, the water levels at the boundary points need to be well in tune. The script from the cht_tide toolbox takes care of this generating the astronomic tide water levels from the tidal constituents  . Happy to explain that more clearly in person